2025-05-19 21:10:37,448 INFO    Thread-6 (_run_job):45591 [wandb_setup.py:_flush():67] Current SDK version is 0.19.7
2025-05-19 21:10:37,449 INFO    Thread-6 (_run_job):45591 [wandb_setup.py:_flush():67] Configure stats pid to 45591
2025-05-19 21:10:37,449 INFO    Thread-6 (_run_job):45591 [wandb_setup.py:_flush():67] Loading settings from /Users/rudra_sarkar/.config/wandb/settings
2025-05-19 21:10:37,449 INFO    Thread-6 (_run_job):45591 [wandb_setup.py:_flush():67] Loading settings from /Users/rudra_sarkar/Documents/PhD IIT Madras/Intro to Deep Learning M Khapra DA6401/da6401_assignment_3/wandb/settings
2025-05-19 21:10:37,450 INFO    Thread-6 (_run_job):45591 [wandb_setup.py:_flush():67] Loading settings from environment variables
2025-05-19 21:10:37,450 INFO    Thread-6 (_run_job):45591 [wandb_init.py:setup_run_log_directory():647] Logging user logs to /Users/rudra_sarkar/Documents/PhD IIT Madras/Intro to Deep Learning M Khapra DA6401/da6401_assignment_3/wandb/run-20250519_211037-isk416ls/logs/debug.log
2025-05-19 21:10:37,451 INFO    Thread-6 (_run_job):45591 [wandb_init.py:setup_run_log_directory():648] Logging internal logs to /Users/rudra_sarkar/Documents/PhD IIT Madras/Intro to Deep Learning M Khapra DA6401/da6401_assignment_3/wandb/run-20250519_211037-isk416ls/logs/debug-internal.log
2025-05-19 21:10:37,451 INFO    Thread-6 (_run_job):45591 [wandb_init.py:init():761] calling init triggers
2025-05-19 21:10:37,451 INFO    Thread-6 (_run_job):45591 [wandb_init.py:init():766] wandb.init called with sweep_config: {'activation': 'tanh', 'batch_size': 1024, 'beam_size': 1, 'dec_dim': 512, 'dec_embedding_dim': 64, 'dropout_rate': 0.6, 'enc_dim': 256, 'enc_embedding_dim': 128, 'learning_rate': 0.008785691224789624, 'linear_dim': 64, 'momentum': 0.8993231423967317, 'n_decoders': 1, 'n_encoders': 1, 'optimizer': 'adamw', 'use_attn': True}
config: {'batch_size': 512, 'native': 'hi', 'model': 'lstm', 'enc_embedding_dim': 128, 'enc_dim': 256, 'n_encoders': 3, 'dec_embedding_dim': 256, 'dec_dim': 128, 'n_decoders': 1, 'linear_dim': 128, 'dropout_rate': 0.2, 'epochs': 20, 'use_attn': True, 'log_connectivity': False, 'momentum': 0.8219466902371052, 'beam_size': 1, 'activation': 'relu', 'optimizer': 'adamax', 'learning_rate': 0.0005253513907244319, 'use_v2': True, 'use_test': False, 'wandb': False, 'wandb_entity': 'da24d008-iit-madras', 'wandb_project': 'da6401-assignment3', 'wandb_sweep': True, 'sweep_id': None, '_wandb': {}}
2025-05-19 21:10:37,451 INFO    Thread-6 (_run_job):45591 [wandb_init.py:init():784] starting backend
2025-05-19 21:10:37,452 INFO    Thread-6 (_run_job):45591 [wandb_init.py:init():788] sending inform_init request
2025-05-19 21:10:37,460 INFO    Thread-6 (_run_job):45591 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2025-05-19 21:10:37,461 INFO    Thread-6 (_run_job):45591 [wandb_init.py:init():803] backend started and connected
2025-05-19 21:10:37,465 INFO    Thread-6 (_run_job):45591 [wandb_run.py:_config_callback():1261] config_cb None None {'activation': 'tanh', 'batch_size': 1024, 'beam_size': 1, 'dec_dim': 512, 'dec_embedding_dim': 64, 'dropout_rate': 0.6, 'enc_dim': 256, 'enc_embedding_dim': 128, 'learning_rate': 0.008785691224789624, 'linear_dim': 64, 'momentum': 0.8993231423967317, 'n_decoders': 1, 'n_encoders': 1, 'optimizer': 'adamw', 'use_attn': True}
2025-05-19 21:10:37,494 INFO    Thread-6 (_run_job):45591 [wandb_init.py:init():896] updated telemetry
2025-05-19 21:10:37,550 INFO    Thread-6 (_run_job):45591 [wandb_init.py:init():920] communicating run to backend with 90.0 second timeout
2025-05-19 21:10:38,219 INFO    Thread-6 (_run_job):45591 [wandb_init.py:init():995] starting run threads in backend
2025-05-19 21:10:39,360 INFO    Thread-6 (_run_job):45591 [wandb_run.py:_console_start():2377] atexit reg
2025-05-19 21:10:39,360 INFO    Thread-6 (_run_job):45591 [wandb_run.py:_redirect():2227] redirect: wrap_raw
2025-05-19 21:10:39,365 INFO    Thread-6 (_run_job):45591 [wandb_run.py:_redirect():2292] Wrapping output streams.
2025-05-19 21:10:39,366 INFO    Thread-6 (_run_job):45591 [wandb_run.py:_redirect():2317] Redirects installed.
2025-05-19 21:10:39,368 INFO    Thread-6 (_run_job):45591 [wandb_init.py:init():1037] run started, returning control to user process
2025-05-19 21:10:58,723 INFO    MainThread:45591 [wandb_run.py:_finish():2112] finishing run da24d008-iit-madras/da6401-assignment3/isk416ls
2025-05-19 21:10:58,726 INFO    MainThread:45591 [wandb_run.py:_atexit_cleanup():2342] got exitcode: 0
2025-05-19 21:10:58,726 INFO    MainThread:45591 [wandb_run.py:_restore():2324] restore
2025-05-19 21:10:58,727 INFO    MainThread:45591 [wandb_run.py:_restore():2330] restore done
2025-05-19 21:10:59,970 WARNING MsgRouterThr:45591 [router.py:message_loop():77] message_loop has been closed
Traceback (most recent call last):
  File "/opt/anaconda3/envs/nlp_env/lib/python3.12/site-packages/wandb/sdk/interface/router_sock.py", line 28, in _read_message
    resp = self._sock_client.read_server_response(timeout=1)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/envs/nlp_env/lib/python3.12/site-packages/wandb/sdk/lib/sock_client.py", line 285, in read_server_response
    data = self._read_packet_bytes(timeout=timeout)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/envs/nlp_env/lib/python3.12/site-packages/wandb/sdk/lib/sock_client.py", line 270, in _read_packet_bytes
    raise SockClientClosedError
wandb.sdk.lib.sock_client.SockClientClosedError

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/opt/anaconda3/envs/nlp_env/lib/python3.12/site-packages/wandb/sdk/interface/router.py", line 70, in message_loop
    msg = self._read_message()
          ^^^^^^^^^^^^^^^^^^^^
  File "/opt/anaconda3/envs/nlp_env/lib/python3.12/site-packages/wandb/sdk/interface/router_sock.py", line 30, in _read_message
    raise MessageRouterClosedError from e
wandb.sdk.interface.router.MessageRouterClosedError
2025-05-19 21:10:59,973 INFO    MsgRouterThr:45591 [mailbox.py:close():115] Closing mailbox, abandoning 2 handles.
